Lib/fontTools/misc/bezierTools.py

```diff
@@ -29,7 +29,19 @@
 
 
 def calcCubicArcLength(pt1, pt2, pt3, pt4, tolerance=0.005):
-    """Return the arc length for a cubic bezier segment."""
+    """Calculates the arc length for a cubic Bezier segment.
+
+    Whereas :func:`approximateCubicArcLength` approximates the length, this
+    function calculates it by "measuring", recursively dividing the curve
+    until the divided segments are shorter than ``tolerance``.
```

Comment:
I don't think this is correct: it stops subdividing once the curve is sufficiently flat, ie. the distance (pt1, pt4) is sufficiently close to the sum of the distances of (pt1, pt2), (pt2, pt3) and (pt3, pt4).

Reply:
Right.